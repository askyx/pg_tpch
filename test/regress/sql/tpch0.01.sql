select tpch_cleanup();
select * from dbgen(0.01);


-- select * from tpch;
-- select * from tpch;
-- select * from tpch(1,1,1);
-- select * from tpch(2,2,2);
