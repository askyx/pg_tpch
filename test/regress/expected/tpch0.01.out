select tpch_cleanup();
 tpch_cleanup 
--------------
 t
(1 row)

select * from dbgen(0.01);
   tab    | row_count 
----------+-----------
 region   |         5
 nation   |        25
 supplier |       100
 customer |      1500
 part     |      2000
 partsupp |      8000
 orders   |     15000
 lineitem |     60175
(8 rows)

-- select * from tpch;
-- select * from tpch;
-- select * from tpch(1,1,1);
-- select * from tpch(2,2,2);
